opensrp-giz-malawi/src/main/java/org/smartregister/giz/util/GizReportUtils.java

```diff
@@ -83,4 +92,70 @@ public static String getStringIdentifier(@NonNull String identifierCode) {
                 .replace(" ", "_")
                 .replace("/", "");
     }
+
+
+    public static ReasonForDefaultingModel getReasonForDefaultingRepository(List<Obs> defaultingObs) throws JsonProcessingException {
```

Comment:
Is this method name right?, It sort of does something else.

Reply:
Named Better